app/code/Magento/Sales/Block/Order/Recent.php

```diff
@@ -55,19 +76,30 @@ public function __construct(
     protected function _construct()
     {
         parent::_construct();
+        $this->getRecentOrders();
+    }
+
+    /**
+     * Get recently placed orders. By default they will be limited by 5.
+     */
+    protected function getRecentOrders()
```

Comment:
Hi @rostyslav-hymon , due to Magento backward compatibility we can't introduce new public and protected methods. Please change it to private.
[Backward compatible development](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)